.storybook/preview-head.html

```diff
@@ -304,6 +304,19 @@
     }
 }
 
+function onPopoverClick(id) {
+    let elementRef = document.getElementById(id);
+    let toggl = elementRef.getAttribute('aria-hidden');
```

Comment:
Is it intentional that `toggle` is `toggl`?

Reply:
typo :)